app/jobs/process_metrics_job.rb

```diff
@@ -7,6 +7,6 @@ class ProcessMetricsJob < ApplicationJob
   ##
   # Delegates the processing to the MetricsDefinitionProcessor service.
   def perform
-    Processors::MetricsDefinition.call
+    Processors::ReviewTurnaround.call
```

Comment:
Shouldn't we change the file name also? For example `process_review_turnaround_job`